src/pages/Tour.tsx

```diff
@@ -84,18 +83,14 @@ const Tour = () => {
       isCentered
       size={step.modalSize}
     >
-      {step.selector ? (
-        <TourOverlay
-          referenceRef={ourRef}
-          padding={spotlightPadding}
-          paddingTop={step.spotlightStyle?.paddingTop}
-          paddingBottom={step.spotlightStyle?.paddingBottom}
-          paddingRight={step.spotlightStyle?.paddingRight}
-          paddingLeft={step.spotlightStyle?.paddingLeft}
-        />
-      ) : (
-        <ModalOverlay />
-      )}
+      <TourOverlay
```

Comment:
I think I did this for a good reason. Will try to dredge out why!

Reply:
Yes, I am worried I'm missing something.

Reply:
It's the MakeCode tour which has a modal on a modal.
Couldn't find an easy way to make that work with the TourOverlay.
A cheat option is to only use ModalOverlay for single step tours with no selector. That just happens to work for us for now until perhaps we sort the modal nature of MakeCode.

Reply:
Hack version here: https://github.com/microbit-foundation/ml-trainer/pull/458